dev/PipsPager/TestUI/PipsPagerExamples.xaml

```diff
@@ -0,0 +1,206 @@
+﻿<Page
+    x:Class="MUXControlsTestApp.PipsPagerExamples"
+    x:Name="PipsPagerExamplesPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:Microsoft.UI.Xaml.Controls"
+    xmlns:local="using:MUXControlsTestApp"
+    xmlns:Windows10FallCreatorsUpdate="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 5)"
+    mc:Ignorable="d">
+
+    <Page.Resources>
+        <ResourceDictionary>
```

Comment:
Are these styles necessary ?

Reply:
@ranjeshj pips defined in the default style are very small and hard to interact with. I thought it might be better to make the pips visuals appear nicer in the usage examples :)

Reply:
@gabbybilka @beervoley The default style should do the right thing and the app should not be requiring a style. If we have an issue with the default styles, we should fix that. Can you remove these custom styles ? We can discuss fixing the interaction issues.

Reply:
If the Pips are clickable, then their size should be 32 x 32 as a minimum touch target - 24 height at a real pinch.

But if the Pips are not interactive, and are just for display - then they can remain smaller.

Reply:
The styles are going to be updated. so I guess this is okay for now.